packages/cli/src/index.ts

```diff
@@ -22,6 +22,7 @@ import { Settings } from './settings';
     .command(commands.New(context))
     .command(commands.Env(context))
     .command(commands.Config(context))
+    .command(commands.SetLang(context))
```

Comment:
should the currently set language be in help text?

Reply:
added - good call